scalyr_agent/util.py

```diff
@@ -127,7 +127,15 @@ def json_dumps_custom(obj, fp):
             else:
                 return json.dumps(obj, sort_keys=True, separators=(",", ":"))
 
-        return lib_name, json_dumps_custom, json.loads
+        if sys.version_info[0] == 3 and sys.version_info[1] < 6:
```

Comment:
LGTM, but can you please also add a test case for this function?

Just a simple call to this function which pass a bytes and string object to it and asserts the result.

Reply:
Forgot about it... Thank for reminding me.

I was thinking of where this test should be located and I chose `json_lib/encode_decode_test.py`

Also, it looks like the `json_lib/encode_decode_test.py` is no longer associated with `json_lib`, Now it is about testing the other JSON libraries for having the same behavior and results. I thought it will be more correct to move it to from `json_lib` to the other tests.

On the other hand, I am not sure should we keep these tests at all.  In many ways, it looks like extra testing of the standard and third-party libraries.

@Kami @czerwingithub, I would be glad to hear your opinion.

Reply:
@ArthurKamalov I think these tests were originally meant to uncover differences between how the third party libs acted.  I'm fine with moving them out, but I think we should keep them, at least the test cases that ensure the libraries behave the same.  We could move them out to as a separate `util_json_test.py` file since it is testing the json shims we have in `util.py`... or just add it as a next test case in `util_test.py` itself.